src/operators/localoperator.jl

```diff
@@ -57,3 +57,17 @@ function Base.repeat(O::LocalOperator, m::Int, n::Int)
     end
     return LocalOperator(lattice, terms...)
 end
+
+function Base.:+(O1::LocalOperator, O2::LocalOperator)
+    if O1.lattice != O2.lattice
```

Comment:
I think we already have a function for this check: `checklattice`.
Could you use that one (or overload it if it does not work already?)

Reply:
This has been done !